src/sas/qtgui/Plotting/Plotter.py

```diff
@@ -253,14 +230,18 @@ def plot(self, data=None, color=None, marker=None, hide_error=False, transform=T
 
         # Now add the legend with some customizations.
         if self.showLegend:
-            width=_legendResize(self.canvas.size().width(), self.parent)
-            if width is not None:
-                self.legend = ax.legend(loc='upper right', shadow=True, prop={'size':width})
-            else:
-                self.legend = ax.legend(loc='upper right', shadow=True)
-            if self.legend:
-                self.legend.set_picker(True)
+            max_legend_width = 30
```

Comment:
I am wondering if we should have this is as setting or is it enough to keep it at 30.

Reply:
I definitely think this could be a setting.  I was pondering setting it dynamically, but 30 works pretty well on most screen sizes / for most reasonable data name lengths.  Another option is to forego the text-wrapping behavior; there is a flag to  `ax.legend()`, something like `expand=True` that makes the legend full width.  It's sort of a question of config bloat vs something that works most of the time.  My feeling was that static-30 is good enough for something that's blocking usability on some platforms.

Reply:
This is now done.